src/Core/Migration/V6_5/Migration1664541794AddIndexForTasks.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_5;
+
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+class Migration1664541794AddIndexForTasks extends MigrationStep
+{
+    public function getCreationTimestamp(): int
+    {
+        return 1664541794;
+    }
+
+    public function update(Connection $connection): void
+    {
+        $connection->executeStatement('ALTER TABLE `cart` ADD INDEX `idx.cart.created_at_updated_at` (`created_at`, `updated_at`)');
+        $connection->executeStatement('ALTER TABLE `cart` ADD INDEX `idx.log_entry.created_at` (`created_at`)');
```

Comment:
why do you need the index only on the `ceated_at` column?

And besides did you experience any slow down for writing and inserting new carts, with this additional indexes? especially with many carts?

Reply:
Because the log_entry.cleanup query only uses the created_at.
cart.cleanup used both.

We have created this patch in our own production environment without any issues. (50.000+ carts)

Reply:
hmm, i did some extra testing with the cart index, it will give preformance issues.
I see someone else fixed the cart delete query with an limit.

Reply:
What performance issues? Can you explain your point a bit more? What needed fixing and was it fixed in shopware or in your production environment?

Reply:
The cart table grew in one day to 10GB (in production env).
Instead of an index there is already a fix in the query with a limit: https://github.com/shopware/platform/commit/64ede82c96f9efe76af4045f61b85a1b10b73248

Reply:
Does that fix make your PR obsolete then? Or does that change just make the second index only on `created_at` obsolet?

And did you have both or just the combined index on that table in your prod env?

Reply:
The extra index on the cart is indeed obsolete, removed it from this PR.
But on the log_entry it is still needed when running the `log_entry.cleanup` handler.

We tested it with both on the cart, an index on created and a combined (created_at, updated_at). These two indexes gave issues.

Reply:
Ah now i understand, sorry but i had trouble as i think there is a slip in your migration, you now still add the index to the `cart` table, but if i have understood you correctly you actually want to add that index to the `log_entry` table only 

Actually you used the correct index name in the migration, but probably just copy and pasted the table name from the first `cart` query

Reply:
My bad, fixed it, and rename the migration.